modules/em/test/test_bayesem3d_ccc.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import print_function
+import IMP
+import IMP.test
+import IMP.em
+import os, sys
+import numpy as np
+import random, math
+
+def get_particles(m, input_pdb):
+    ### Read input fragment
+    sel0 = IMP.atom.BackbonePDBSelector()
+    sel1 = IMP.atom.CBetaPDBSelector()
+    mh = IMP.atom.read_pdb(input_pdb, m, IMP.atom.OrPDBSelector(sel0, sel1))
+    IMP.atom.add_radii(mh)
+    return mh, IMP.core.get_leaves(mh)
+
+class Tests(IMP.test.TestCase):
+    """Derivative score stability Test"""
+
+    def setUp(self):
+
+        IMP.test.TestCase.setUp(self)
+
+
```

Comment:
You have a few seemingly random blank lines here. If nothing else, `flake8` will complain about them.